conn.go

```diff
@@ -591,6 +591,8 @@ func (cn *conn) simpleExec(q string) (res driver.Result, commandTag string, err
 	b.string(q)
 	cn.send(b)
 
+	res = emptyRows
```

Comment:
While I agree from a spec-compliant protocol perspective, `res` will be `nil` if, for some reason, a `Z` is sent before an `I`, yielding the same panic. It is possible we should set `err` to something and then clear it when either `I` or `C` occur. This would send an error in any case where res wasn't set. Agree?